.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Python CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches:
+    - '**'
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ['3.8']
+        toxenv: [django22-celery44, django22-celery50, django30-celery50, django31-celery50, django32-celery50,
+          quality, docs, pii_check]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Upgrade pip
+      run: pip install --upgrade pip
```

Comment:
why do we need to upgrade pip everytime, it should be installing latest version from `pip.txt`